src/antora-adapter.js

```diff
@@ -17,6 +17,7 @@ module.exports = (file, contentCatalog, vfs) => {
             module,
             family: 'image',
             mediaType: image.mediaType,
+            path: image.relative + '/' + image.basename,
```

Comment:
We should probably use the module `path` to join the path:
```suggestion
            path: path.join(image.relative, image.basename),
```